hazelcast/src/main/java/com/hazelcast/util/executor/CompletableFutureTask.java

```diff
@@ -89,4 +89,11 @@ protected void cancelled(boolean mayInterruptIfRunning) {
         }
     }
 
+    @Override
+    public String toString() {
+        return "CompletableFutureTask{"
+                + "callable=" + callable
```

Comment:
Can you clean up the toString so that either callable or runnable is included? No need to include both, since one of them is never set.